code/drasil-example/swhs/lib/Drasil/SWHS/References.hs

```diff
@@ -1,6 +1,4 @@
-module Drasil.SWHS.References (citations, bueche1986, incroperaEtAl2007,
-  koothoor2013, lightstone2012, parnasClements1986, smithEtAl2007,
-  smithLai2005, smithKoothoor2016) where
+module Drasil.SWHS.References where
```

Comment:
why remove the explicit list of exports?

Reply:
In all honesty, that was just laziness w.r.t. sorting them out. Some of these exports were re-exports of things from `drasil-data`. I can add them back.

Regarding codestyle, unless there's a reason to manually write the export lists, isn't "export all by default" okay for modules like this one, where everything is exported anyways (and where we expect all future content to be too)?

Reply:
If you leave a comment saying "exports everything, on purpose", then that's fine.